roles/sync_plans/README.md

```diff
@@ -0,0 +1,58 @@
+theforeman.foreman.sync_plans
+=============================
+
+This role defines Sync Plans.
+
+Role Variables
+--------------
+
+This role supports the [Common Role Variables](https://github.com/theforeman/foreman-ansible-modules/blob/develop/README.md#common-role-variables).
+
+- `sync_plans`: List of sync plans to create. Each sync plan is represented as a dictionary which specifies the `name` of the sync plan and the `products` assigned to the sync plan. It also specifies the `interval` which can be 'hourly', 'daily', 'weekly', or 'custom cron'. In case the 'custom cron' `interval` is used, it should also specify the `cron_expression`. Finally the sync plan should have a `sync_date` which specifies the first time that the sync plan will run.
```

Comment:
I wonder how much this should duplicate the `sync_plan` module docs.

Reply:
My general sense is that the role docs should be complete enough so that the average user could avoid using the modules or their documentation at all in cases where a role exists.

Reply:
I always struggle to find the right balance between "this is required" and "a copy that gets out of date soon"